content/events/2018/tech-day-02-17.md

```diff
@@ -0,0 +1,17 @@
+---
+title: February Tech Day 2
+date: 2018-02-17
+---
+
+{{< location 25 >}}
+
+The day will run from 10:00am until 5:00pm, however feel free to come and go as you please.
+
+## Food
+Food on campus is scarce on weekends, so we recommend either ordering in food, or bringing something with you. There are a few corner-shops located nearby however. There's a [map of local food places](https://goo.gl/yYlfs5) to help you find nearby food.
+
+## Electronics
+As expected, you'll need to bring your robot to work on, in whatever state it may be. We also recommend bringing at least 1 laptop with you to work on. Laptops will be provided for you to use.
+
+## Supervision
+Team members must be accompanied by any adult who is able to take responsibility for all of them. (this would ideally be a teacher, but it can be a parent if all the other parents agree).
```

Comment:
Even though "(this would ideally" is in parens, I think it should still start with a capital letter. (Also in the above duplicate)